frontend/public/components/source-to-image.tsx

```diff
@@ -89,18 +89,25 @@ class BuildSource extends React.Component<BuildSourceProps, BuildSourceState> {
     };
   }
 
+  componentDidMount() {
```

Comment:
`componentDidUpdate()` is not called on initial render, so need this to update the tags.